README.md

```diff
@@ -1,2 +1,16 @@
 # photo-converter
-Service that expose a RESTful API to convert JPEG to PNG and vice versa and compress the image with the compression ratio specified by the user.
+Service that expose a RESTful API to convert JPEG to PNG and vice versa and compress the image 
+with the compression ratio specified by the user. The user has the ability to view
+the history and status of their requests (for example, queued, processed, completed) and upload 
+the original image and the processed one.
+# Architecture Diagram
+![alt text](./docs/architecture-diagram.png)
+# Endpoints
+- /user/requests - view user requests history [GET]
```

Comment:
We can leave it as is, but logically requests is more a list of conversions, so maybe it should not be under users path

Reply:
Interesting remark, thanks